src/rpc/webrtc.rs

```diff
@@ -300,14 +300,12 @@ pub(crate) async fn action_with_timeout<T>(
         let f = f;
     }
 
-    loop {
```

Comment:
@stuqdog clippy was erroring because of this `loop`. It looks like this error was only happening with the most recent version of clippy (I wasn't seeing it locally until a ran `rustup update && cargo clippy --all-features`). The linter is complaining because this `loop` is hypothetically unnecessary, as both branches of the `tokio::select!` return a `Result` and end the function, so the loop will never actually iterate.